### DISCUSSION TITLE
Feature: map keys for different programs

### DISCUSSION THREAD

**Discussion by mikdmst1** [sezanzeb/input-remapper#251] [Ideas]
hi! I want map mouse keys only for Blender now, not globally. can u do this Feature? Thanks!

**Comment by sezanzeb** [comment]
It wont't work in wayland, Also see https://github.com/sezanzeb/input-remapper/issues/32, https://github.com/sezanzeb/input-remapper/issues/56, https://github.com/sezanzeb/input-remapper/issues/203, https://github.com/sezanzeb/input-remapper/discussions/20 and https://github.com/sezanzeb/input-remapper/discussions/233

**Comment by mikdmst1** [comment]
i dont need wayland today :)
ok thanks lets see some scripts from users, thx!
